### PR TITLE
Upgrade to ns-3.33

### PR DESCRIPTION
**Pull request by kschrab**
## Type of this PR 

- [ ] Bug fix
- [x] Enhancement

## Description

Upgrade ns-3 federate to ns-3.33

* Fixed compile errors in ns3-federate (see https://github.com/mosaic-addons/ns3-federate/pull/1)
* Various attributes for the models have been renamed (see changes in `ns3_federate_config.xml`)
  * [ ] `ns3::QueueBase::MaxPackets` is not available anymore. Instead one should use `ns3::QueueBase::MaxSize`. However, this attribute expects an object and I did not yet find a way to instantiate such object via XML. According to https://www.nsnam.org/docs/manual/html/attributes.html#config-setdefault-and-commandline you simply could use `"400p"`, but that does not work.
  * [ ] Almost every attribute from `WifiMac` have been moved to `WifiPhy`. Some of them seem to have no replacing attribute.
  * [ ] Some attributes from `WifiPhy`, such `ShortGuardEnabled` or `GreenfieldEnabled`, have been moved to a separate `HtConfiguration`. Since they were disabled anyhow, I did not added a replacement for that yet.
* The `omnet_ns3_txpower` test in our internal pipeline does not succeed: there's one message which does not arrive anymore. Could be related to the unfinished migration of the ns3_config or to something completely different.

## Issue(s) related to this PR

Internal issue 146 
 
## Affected parts of the online documentation

- Adjust version in online documentation

## Definition of Done

- [x] You have read CONTRIBUTING.md carefully.
- [x] You have signed the [Contributor License Agreement](http://www.eclipse.org/legal/CLA.php).
- [x] All commits are signed-off (`-s`) with your mail address of your Eclipse Account.
- [x] `origin/main` has been merged into your Fork.
- [x] New functionality is covered by unit tests or integration tests. Code coverage must not decrease.
- [x] There are no new SpotBugs warnings. 
- [x] Coding guidelines have been observed (see CONTRIBUTING.md).
- [ ] All tests pass.

### CODE REVIEW

**Comment by kschrab** [comment]
@fhilg please review, the test `omnet_ns3_txpower` currently fails in our pipeline due to a missing message

**Comment by kschrab** [comment]
We would need to add this to the `logback.xml`:

```xml
<logger name="Ns3AmbassadorOutput" additivity="false" level="INFO">
    <appender-ref ref="CommunicationDetailsLog"/>
</logger>
<logger name="Ns3AmbassadorError" additivity="false" level="ERROR">
    <appender-ref ref="CommunicationDetailsLog"/>
</logger>
<logger name="OmnetppAmbassadorOutput" additivity="false" level="INFO">
    <appender-ref ref="CommunicationDetailsLog"/>
</logger>
<logger name="OmnetppAmbassadorError" additivity="false" level="ERROR">
    <appender-ref ref="CommunicationDetailsLog"/>
</logger>
```

and replace the respective lines in mosaic-extended accordingly.

File: bundle/src/assembly/resources/fed/ns3/ns3_installer.sh [hunk 1]
```diff
@@ -69,8 +69,8 @@ ns3_scratch="${ns3_simulator_folder}/scratch"
 ns3_source="${ns3_simulator_folder}/src"
 
 ####### semi automatic parameters ########
-ns3_federate_url="https://github.com/mosaic-addons/ns3-federate/archive/21.0.zip"
-ns3_url="http://www.nsnam.org/release/$ns3_version_affix.tar.bz2"
+ns3_federate_url="https://github.com/mosaic-addons/ns3-federate/archive/refs/heads/ns-3-33.zip"
```
Comment (rprotzmann) [inline bundle/src/assembly/resources/fed/ns3/ns3_installer.sh:72]: Now already **ns-3-34.zip**

**Review by rprotzmann — APPROVED** [review]
Approved, when final change is tested

**Review by rprotzmann — APPROVED** [review]

**Pull request merged**